pi4j-core/src/main/java/com/pi4j/boardinfo/util/BoardInfoHelper.java

```diff
@@ -64,6 +64,10 @@ public static boolean usesRP1() {
         return instance.boardInfo.getBoardModel() == BoardModel.MODEL_5_B;
     }
 
+    public static boolean runsOnRaspberryPi() {
```

Comment:
maybe we want to name it differently: runningOnRaspberryPi() ?

Reply:
good point! will change

Reply:
is done